appengine/test/tc_base_image.rb

```diff
@@ -0,0 +1,18 @@
+require "minitest/autorun"
+require "#{::File.dirname(__FILE__)}/test_helper.rb"
+
+
+# Basic tests on the content of the base image
+
+class TestBaseImage < ::Minitest::Test
+
+  HELPER = TestHelper.new(verbose: ::ENV['VERBOSE'])
```

Comment:
This is not picking up the global set in the Rakefile.

Why is the helper instance constant?

If all the tests use a helper like this, we could have a test class that sets this up, something like:

```ruby
class Test < Minitest::Test
  def helper
    @helper ||= Helper.new(ENV['VERBOSE'])
  end
end
```